src/casp/services/_settings.py

```diff
@@ -23,6 +23,12 @@ class AllEnvSettings(BaseSettings):
     SERVICES_DIR = SERVICES_DIR
     DEFAULT_SERVICE_HOST: str = "0.0.0.0"
     DEFAULT_SERVICE_PORT: int = 8000
+    API_SERVICE_PORT: int = DEFAULT_SERVICE_PORT
+    MODEL_SERVICE_PORT: int = DEFAULT_SERVICE_PORT
+    LOG_LEVEL: str = "info"
```

Comment:
I don't know if you added any debug logging in this PR, so this question might be irrelevant for now anyway

Reply:
Yeah, I tend to use info as debug.  Especially here where we don't have that many log messages. If it gets chatty, we can certainly lower a bunch of the logs to debug.  That might make more logs since libraries might log more stuff